CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ A [separate changelog is kept for rand_core](rand_core/CHANGELOG.md).
 
 You may also find the [Upgrade Guide](https://rust-random.github.io/book/update.html) useful.
 
+## [Unreleased]
+- Derive PartialEq+Eq for StdRng, SmallRng, ThreadRng, ReseedingRng, ReseedingCore, and StepRng (#975)
```

Comment:
It doesn't make any sense for `ThreadRng` because that's just a handle. Handles cannot cross thread boundaries so any two handles which can be compared should be equal. I'm also not sure it's useful on the reseeding constructs.